plugins/ufm_events_grafana_dashboard_plugin/src/ufm_events_grafana_dashboard/prometheus/prometheus.proto

```diff
@@ -0,0 +1,81 @@
+// Copyright 2016 Prometheus Team
```

Comment:
Why do we need the prometheus proto and py code directly in the repo? 
Haven't we installed it in the docker file?
I might be missing something here :)

Reply:
We installed the Prometheus server.
Our python application is the client that sends data to the server in the Remote Write protocol.

"The remote write request MUST be encoded using Google Protobuf 3, and MUST use the schema defined above. Note [the Prometheus implementation](https://github.com/prometheus/prometheus/blob/v2.24.0/prompb/remote.proto) uses [gogoproto optimisations](https://github.com/gogo/protobuf)"

The current .proto is shared between all plugins that use this protocol, in the future we're going to have centralized Prometheus server that will be shared between all clients.

I added the URL for the original file in their repo